routes/valueRouter.js

```diff
@@ -29,11 +68,42 @@ valueRouter.get('/:itemId', async (req, res) => {
 // CREATE a new item
 valueRouter.post('/', async (req, res) => {
   try {
-    const { item_name: itemName, quantity_type: quantityType, price, category } = req.body;
-    await db.query(
+    const { itemName, quantityType, price, category } = req.body;
+    const item = await db.query(
       'INSERT INTO fair_market_value (item_name, quantity_type, price, category) VALUES ($(itemName), $(quantityType), $(price), $(category)) RETURNING *',
       { itemName, quantityType, price, category },
     );
+    res.status(200).send(item);
+  } catch (err) {
+    res.status(500).send(err.message);
+  }
+});
+
+// edit item
+valueRouter.put('/:id', async (req, res) => {
```

Comment:
🚫 **[eslint]** <[consistent-return](https://eslint.org/docs/latest/rules/consistent-return)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected to return a value at the end of async arrow function.